org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/JavaLaunchableTester.java

```diff
@@ -144,17 +144,17 @@ public class JavaLaunchableTester extends PropertyTester {
 	 */
 	private IType getType(IJavaElement element) {
         IType type = null;
-        if (element instanceof ICompilationUnit) {
-            type= ((ICompilationUnit) element).findPrimaryType();
+		if (element instanceof ICompilationUnit iCompilationUnit) {
```

Comment:
on github the intendation looks wrong here

Reply:
in local its ok
<img width="671" alt="image" src="https://github.com/user-attachments/assets/9cda801a-78b5-465a-9e61-d49eef08de6e">

Reply:
Will do this.